modules/google/helm-dependencies.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
```

Comment:
Here you should just symlink the file from the root module like it is done in module/aws